services/ui-src/src/components/pages/Dashboard/DashboardPage.tsx

```diff
@@ -162,13 +168,33 @@ export const DashboardPage = ({ reportType }: Props) => {
           planTypeIncludedInProgram: report?.planTypeIncludedInProgram,
           "planTypeIncludedInProgram-otherText":
             report?.["planTypeIncludedInProgram-otherText"],
+          newOrExistingProgram: report.newOrExistingProgram,
```

Comment:
won't this be undefined for MLR and NAAAR? I'm actually surprised it doesn't throw an error in console

Reply:
Youre right, and its actually been doing just fine handling these things in the past as `combinedData: report.combinedData,` is being passed just like this. I'll make this as optionally passed though to be clear it really could be!